esp/esp/program/modules/handlers/studentclassregmodule.py

```diff
@@ -776,7 +776,7 @@ def openclasses(self, request, tl, one, two, module, extra, prog):
 
         module = prog.getModule('OnSiteClassList')
         if module:
-            return module.classList_base(request, tl, one, two, module, 'by_time', prog, 'allclass_fragment.html')
+            return module.classList_base(request, tl, one, two, module, 'by_time', prog, options={}, template_name='allclass_fragment.html')
```

Comment:
Passing `None` takes you to a landing page for choosing the options, which isn't what we want here because this one uses a different template which doesn't use those options anyway.